hack/dockerized

```diff
@@ -33,11 +33,26 @@ trap finish EXIT
 
 RSYNCD_PORT=$(docker port $RSYNC_CID 873 | cut -d':' -f2)
 
-while ! rsync ${KUBEVIRT_DIR}/${RSYNCTEMP} "rsync://root@127.0.0.1:${RSYNCD_PORT}/build/${RSYNCTEMP}"; do
-    echo "Waiting for rsyncd to be ready"
-    sleep .1
+rsynch_fail_count=0
+
+while ! rsync ${KUBEVIRT_DIR}/${RSYNCTEMP} "rsync://root@127.0.0.1:${RSYNCD_PORT}/build/${RSYNCTEMP}" &>/dev/null; do
```

Comment:
bash has a `SECONDS` environment variable. You could set SECONDS to zero before you start the loop and just exit once SECONDS contains a value greater than 30. The auto-increment of SECONDS is a bash feature.

Reply:
also you can always use a sleep of 0.1. How long the rsync call will block is not easy to say. So in your implementation it may as well try for 60 seconds to connect.

Reply:
i want to give indication that the system is not stuck, so i print the dots. dont think we should do that every 100ms

Reply:
regarding SECONDS, it makes things more complex in the first check (the one that should print "Waiting for rsyncd to be ready"). do you think it is critical?